### DISCUSSION TITLE
Seek Buttons on CDN

### DISCUSSION THREAD

**Discussion by kaambiz** [vidstack/player#955] [Q&A]
I cannot seem to figure out how to show them on V1.1.11 as there are no documentation for them as of now. I followed the old method which is working on 0.6 but it's a no-show:

```
<media-provider></media-provider>
<media-seek-button seconds="+30">
  <media-icon type="seek-forward"></media-icon>
</media-seek-button>

<media-seek-button seconds="-30">
  <media-icon type="seek-backward"></media-icon>
</media-seek-button>
 <media-video-layout></media-video-layout>
```

**Comment by mihar-22** [comment]
You haven't applied the default theme (`class="vds-button"`). Everything is headless so classes need to be applied. Check out the [default theme examples](https://github.com/vidstack/examples).

```html
<media-seek-button class="vds-button" seconds="+30">
  <media-icon type="seek-forward"></media-icon>
</media-seek-button>
```

**Reply by kaambiz** [reply]
Ok, I'm more confused now :)
I'm using the default layout and adding just the seek buttons as shown in examples would result in this:
 
![127 0 0 1_5500_wc-test html](https://github.com/vidstack/vidstack/assets/59145/e0c85320-d26f-4e85-b08b-e63c23a18ee0)

Here is the sample html:



```html
<body>
        <main>
        <media-player title="Tears of Steel" stream-type="live:dvr" src="https://cph-p2p-msl.akamaized.net/hls/live/2000341/test/master.m3u8">
            <media-provider></media-provider>
            <media-seek-button class="vds-button" seconds="+30">
                <media-icon type="seek-forward"></media-icon>
            </media-seek-button>
            <media-video-layout></media-video-layout>
        </media-player>
        
        <script
        src="https://cdn.jsdelivr.net/npm/vidstack@next/cdn/with-layouts/vidstack.js"
        type="module"
        ></script>
    </main>
    </body>
```


Everything works fine on Default Layout except seek buttons. Do I need to build a player using the headless option?

**Reply by mihar-22** [reply]
Ye you can't add controls like that inside the player, it's still all just HTML. Nothing magical can happen here to position that button for you. If you want to customize consider building your own using the examples to guide you. You can extend the default theme, use CSS, or Tailwind CSS. All the examples are there in the vidstack/examples repo.

**Reply by kaambiz** [reply]
Oh I see. Default layout has no styles for seek buttons. Gotcha.

Thanks.
